src/open_inwoner/accounts/migrations/0043_change_digid_email.py

```diff
@@ -0,0 +1,30 @@
+from django.db import migrations
+from open_inwoner.utils.hash import generate_email_from_string
+from ..choices import LoginTypeChoices
+
+
+def change_bsn_email_to_hash(apps, _):
+    User = apps.get_model("accounts", "User")
+
+    for user in User.objects.filter(login_type=LoginTypeChoices.digid).all():
+        if user.email == f"user-{user.bsn}@bsn.com":
+            user.email = generate_email_from_string(user.bsn)
+            user.save()
+
+
+def change_hash_email_to_bsn(apps, _):
+    User = apps.get_model("accounts", "User")
+
+    for user in User.objects.filter(login_type=LoginTypeChoices.digid).all():
+        if user.email == generate_email_from_string(user.bsn):
```

Comment:
In this PR I change how we generate an email for Digid users: before it was generated like `user-{user.bsn}@bsn.com`, but it's not a good thing to expose the user's BSN in the email, that's why I replace it with hash (this hash is not secure, but the email would be used temporarily before the user fills in the correct one, so it should do). 

This file is a data migration which replaces the old generated emails with the new ones and this function does the reverse thing: replace the hashed email with `user-{user.bsn}@bsn.com` email, since it is used when you are making a reverse migration.

Reply:
I've explained this to Vasileios, so should be cleared up now.

I would like to replace 'bsn.com' with 'example.com'. This to ensure that we don't send email by accident to a domain which isn't under our control.